pkg/reconciler/taskrun/resources/pod_test.go

```diff
@@ -724,128 +724,6 @@ func TestMakeAnnotations(t *testing.T) {
 	}
 }
 
-func TestInitOutputResourcesDefaultDir(t *testing.T) {
-	names.TestingSeed()
-
-	randReader = strings.NewReader(strings.Repeat("a", 10000))
-	defer func() { randReader = rand.Reader }()
-
-	trs := v1alpha1.TaskRunSpec{
-		Outputs: v1alpha1.TaskRunOutputs{
-			Resources: []v1alpha1.TaskResourceBinding{{
-				PipelineResourceBinding: v1alpha1.PipelineResourceBinding{
-					Name: "outputimage",
-					ResourceRef: v1alpha1.PipelineResourceRef{
-						Name: "outputimage",
-					},
-				},
-			}},
-		},
-	}
-	ts := v1alpha1.TaskSpec{
-		Steps: []v1alpha1.Step{{Container: corev1.Container{
-			Name:  "task-with-output-image",
-			Image: "image",
-		}}},
-		Outputs: &v1alpha1.Outputs{
-			Resources: []v1alpha1.TaskResource{{
-				ResourceDeclaration: v1alpha1.ResourceDeclaration{
-					Name: "outputimage",
-				},
-				OutputImageDir: fmt.Sprintf("%s/outputimage", v1alpha1.TaskOutputImageDefaultDir),
-			}},
-		},
-	}
-	annotations := map[string]string{
-		"simple-annotation-key": "simple-annotation-val",
-	}
-	want := &corev1.PodSpec{
-		RestartPolicy: corev1.RestartPolicyNever,
-		InitContainers: []corev1.Container{{
-			Name:         containerPrefix + credsInit + "-9l9zj",
-			Image:        credsImage,
-			Command:      []string{"/ko-app/creds-init"},
-			Args:         []string{},
-			Env:          implicitEnvVars,
-			VolumeMounts: implicitVolumeMounts,
-			WorkingDir:   workspaceDir,
-		}, {
-			Name:         "create-dir-default-image-output-mz4c7",
-			Image:        "override-with-bash-noop:latest",
-			Command:      []string{"/ko-app/bash"},
-			Args:         []string{"-args", "mkdir -p /builder/home/image-outputs/outputimage"},
```

Comment:
question: do we not still want this test but with the new path? or do you we feel like this is already covered somewhere else/

Reply:
The function that this was testing got deleted, it now uses the normal output directory creation stuff, which is tested: https://github.com/tektoncd/pipeline/blob/master/pkg/reconciler/taskrun/resources/output_resource_test.go#L171